lib/LaTeXML/Util/Test.pm

```diff
@@ -235,7 +235,11 @@ sub daemon_ok {
   $invocation .= " 2>$localname.test.status ";
   if (!$generate) {
     pathname_chdir($dir);
-    is(system($invocation), 0, "latexmlc invocation for test $localname: $invocation yeilded $!");
+    my $exit_code = system($invocation);
+    if ($exit_code != 0) {
+      $exit_code = $exit_code >> 8;
```

Comment:
This is an interesting change I noticed was needed - sometimes the return code on successful runs was shown as 255 or 256 instead of 0 and 1, also something we have seen in some of the CPANTester runs, and I believe it has something to do with the inner mechanics of how `waitpid` passes children exit values (as for the `system` call in the daemon tests). A link I half-understand:

https://stackoverflow.com/a/3659648

In any case, the only case this affects in this part of the code is recognizing 255 as a 0 exit, so that the test can succeed in that setup.